codespell_lib/data/dictionary.txt

```diff
@@ -20104,6 +20119,7 @@ mocule->module
 mocules->modules
 moddel->model
 moddels->models
+modelinng->modeling
```

Comment:
```suggestion
modelinng->modeling
modellinng->modelling
```
See https://www.grammarly.com/blog/modeling-or-modelling/

Do you want to add to the GB/US dict too?